package.json

```diff
@@ -37,9 +40,12 @@
     "shelljs": "~0.3.0"
   },
   "devDependencies": {
+    "chai": "^2.0.0",
```

Comment:
Unfortunately I just realized that chai is [not working on IE 8](https://github.com/chaijs/chai/issues/117) and cannot be used in Riot. Bummer!